src/helpers/spacing.test.js

```diff
@@ -67,7 +110,7 @@ describe('@mixin govuk-responsive-spacing', () => {
             padding-top: 25px; } }`)
   })
 
-  it('throws an exception when passed anything other than a map', async () => {
+  it('throws an exception when passed anything other than a number', async () => {
```

Comment:
Technically '14px' is a number in Sass, so maybe 'throws an exception when passed a non-existent point' or similar?